cobalt/browser/migrate_storage_record/migration_manager.h

```diff
@@ -15,12 +15,47 @@
 #ifndef COBALT_COBALT_MIGRATE_STORAGE_RECORD_MIGRATION_MANAGER_H_
 #define COBALT_COBALT_MIGRATE_STORAGE_RECORD_MIGRATION_MANAGER_H_
 
+#include <atomic>
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "base/functional/bind.h"
+#include "content/public/browser/weak_document_ptr.h"
 #include "content/public/browser/web_contents.h"
+#include "net/cookies/canonical_cookie.h"
 
 namespace cobalt {
 namespace migrate_storage_record {
 
-void DoMigrationTasksOnce(content::WebContents* web_contents);
+using Task = base::OnceCallback<void(base::OnceClosure)>;
+using Tasks = std::vector<Task>;
+
+template <typename T>
+using ItemPtr = std::unique_ptr<T>;
+template <typename T>
+using ItemPtrs = std::vector<ItemPtr<T>>;
+
+using StringPair = std::pair<std::string, std::string>;
```

Comment:
I feel conflicted about renaming this many data types, because when reading the test code I had to keep coming back here to see what they were, which points to an abstraction that lacks a purpose. Perhaps the first one (`Task`) makes sense (and it should be inside the class that uses it, and not outside), but the others I feel like they are just syntactic sugar and add friction to the code reading. WDYT?